src/net/sourceforge/plantuml/ugraphic/ImageBuilder.java

```diff
@@ -281,13 +281,15 @@ private Image getAviImage(AffineTransformation affineTransform) throws IOExcepti
 	private UGraphic2 createUGraphic(FileFormatOption option, long seed, final Dimension2D dim, Animation animationArg,
 			double dx, double dy) {
 		final ColorMapper colorMapper = param.getColorMapper();
+		final double scaleFactor = (param.getScale() == null ? 1 : param.getScale().getScale(dim.getWidth(), dim.getHeight()))
+				* param.getDpi() / 96.0;
```

Comment:
Is this calculation correct?

Reply:
Well, it _sounds_ correct. Now it's difficult to be 100% affirmative without testing.
However, I think you can go ahead for it. We will fix it if needed :-)